### DISCUSSION TITLE
TBD Version 1.0.0 Release Candidate 1

### DISCUSSION THREAD

**Discussion by ctag-fh-kiel** [ctag-fh-kiel/ctag-tbd#69] [Announcements]
This TBD firmware release has many improvements and new features. 
Stability should be greatly improved compared to the previous versions.
Details will be posted in versions.md once v1.0.0 will be released. 
Please comment in the discussion forum and file any issues.

Enjoy!

<hr /><em>This discussion was created from the release <a href='https://github.com/ctag-fh-kiel/ctag-tbd/releases/tag/1.0.0_rc1'>TBD Version 1.0.0 Release Candidate 1</a>.</em>

**Comment by schrab** [comment]
Such a great news!

**Reply by machinehistories** [reply]
Is the tbd-bba going to be available for purchase. I read over the midi documentation and it is so deeply developed and I really look forward to using it. Do you still have plans for releasing this device and or the tbd mk2. In the v1 firmware it shows 8 cv's does that mean v1 cv input can be expanded and if so how? Thanks

**Comment by ctag-fh-kiel** [comment]
Hi there :) We are currently working on a tbd-bba version, which will be a midi only version of the tbd, offering plenty of parameter access and modulation options. My feeling is that by end of the year it could be available for purchase.

As to the tbd mk2, which is a much enhanced eurorack version including many more CVs, pots, trigs etc, I think it will only be available as DIY. Would you be interested? 

Cheers and thanks :)

**Reply by schrab** [reply]
The bba version idea was in the air, I've started working on standalone box version couple years ago but now have no access to my lab.
MIDI control is the best way to keep hardware simple but some user interface will allow to use the device in dawless setup along with other gear. Nts-1 is a good example but lacks of flexibility unlike TBD ;)

**Reply by ctag-fh-kiel** [reply]
Thanks for your feedback, yes, as far as I can tell from my testing with the current prototype, TBD works great as MIDI only box. I hope I can soon tell you more about it.

**Comment by Moa13** [comment]
Thanks for the update! is there a change log anywhere to be found?

**Reply by ctag-fh-kiel** [reply]
https://github.com/ctag-fh-kiel/ctag-tbd/blob/master/versions.md

**Reply by Moa13** [reply]
> https://github.com/ctag-fh-kiel/ctag-tbd/blob/master/versions.md

And one more if I may, where do I acces the MIDI web GUI ? I guess it's somewhere else then http://ctag-tbd.local ? as I can't see any MIDI assignments there.

**Comment by ctag-fh-kiel** [comment]
What TBD are you using? MIDI only works for the to be released **non eurorack** version.

**Comment by ctag-fh-kiel** [comment]
Version 1.0.0 is now released. It is effectively the RC1.
